tests/integration/conftest.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Fixtures for the SAML Integrator charm integration tests."""
+
+from pathlib import Path
+
+import ops
+import pytest_asyncio
+import yaml
+from pytest import fixture
+from pytest_operator.plugin import OpsTest
+
+
+@fixture(scope="module", name="metadata")
+def metadata_fixture():
+    """Provide charm metadata."""
+    yield yaml.safe_load(Path("./metadata.yaml").read_text("utf-8"))
```

Comment:
Can we inline the `metadata` fixture, since it's only used once in `app_name_fixture`.

Reply:
Inline?

Reply:
Oh, I mean just put the metadata loading in the `app_name_fixture`

Reply:
Makes sense